components/Trip/Trip.styled.js

```diff
@@ -0,0 +1,27 @@
+import { styled } from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+
+export const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  gap: 10px;
+  background-color: #fff;
+  padding: 12px;
+  border-radius: 10px;
+  border: 1px solid rgba(68, 140, 255, 0.15);
+`;
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`;
```

Comment:
Does having 8,12,10...different px values intended for design reasons?

I would use a more consistent `rem` value of 1 or 0.75 all over the place, to keep all the spacing in beat.

Reply:
The px values are there for design reasons, since we have a fully flashed out design in Figma that we use as a guide for spacing. 🐸